api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -440,159 +449,153 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	
 	/**
-	 * @see ConceptService#getDrugs(String, Concept, boolean, boolean, boolean, Integer, Integer)
+	 * @see org.openmrs.api.ConceptService#getDrugs(String, Concept, boolean, boolean, boolean, Integer, Integer)
 	 */
 	public List<Drug> getDrugs(String drugName, Concept concept, boolean searchOnPhrase, boolean searchDrugConceptNames,
 	        boolean includeRetired, Integer start, Integer length) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getDrugsByIngredient(Concept)
+	 * @see org.openmrs.api.ConceptService#getDrugsByIngredient(Concept)
 	 */
 	public List<Drug> getDrugsByIngredient(Concept ingredient);
 	
 	/**
-	 * @see ConceptService#getConceptMapTypes(boolean, boolean)
+	 * @see org.openmrs.api.ConceptService#getConceptMapTypes(boolean, boolean)
 	 */
 	public List<ConceptMapType> getConceptMapTypes(boolean includeRetired, boolean includeHidden) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptMapType(Integer)
+	 * @see org.openmrs.api.ConceptService#getConceptMapType(java.lang.Integer)
 	 */
 	public ConceptMapType getConceptMapType(Integer conceptMapTypeId) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptMapTypeByUuid(String)
+	 * @see org.openmrs.api.ConceptService#getConceptMapTypeByUuid(java.lang.String)
 	 */
 	public ConceptMapType getConceptMapTypeByUuid(String uuid) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptMapTypeByName(String)
+	 * @see org.openmrs.api.ConceptService#getConceptMapTypeByName(java.lang.String)
 	 */
 	public ConceptMapType getConceptMapTypeByName(String name) throws DAOException;
 	
 	/**
-	 * @see ConceptService#saveConceptMapType(ConceptMapType)
+	 * @see org.openmrs.api.ConceptService#saveConceptMapType(org.openmrs.ConceptMapType)
 	 */
 	public ConceptMapType saveConceptMapType(ConceptMapType conceptMapType) throws DAOException;
 	
 	/**
-	 * @see ConceptService#purgeConceptMapType(ConceptMapType)
+	 * @see org.openmrs.api.ConceptService#purgeConceptMapType(org.openmrs.ConceptMapType)
 	 */
 	public void deleteConceptMapType(ConceptMapType conceptMapType) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptReferenceTerms(boolean)
+	 * @see ConceptService#getConceptReferenceTerms(java.lang.boolean)
 	 */
 	public List<ConceptReferenceTerm> getConceptReferenceTerms(boolean includeRetired) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptReferenceTerm(Integer)
+	 * @see ConceptService#getConceptReferenceTerm(java.lang.Integer)
 	 */
 	public ConceptReferenceTerm getConceptReferenceTerm(Integer conceptReferenceTermId) throws DAOException;
 	
 	/**
-	 * @see ConceptService#getConceptReferenceTermByUuid(String)
+	 * @see ConceptService#getConceptReferenceTermByUuid(java.lang.String)
 	 */
 	public ConceptReferenceTerm getConceptReferenceTermByUuid(String uuid) throws DAOException;
 	
+	/**
+	 * Gets a list of concept reference terms with a specific ConceptSource
+	 * 
```

Comment:
Same comment here as above.

Reply:
@dkayiwa This function is not found in ConceptService.java. I have been instructed by @samuelmale to simply document them, based on their function in HibernateConceptDAO.java. :)